### PR TITLE
adds stormtrooper armor and helmet and a cargo crate for them

### PR DESCRIPTION
**Pull request by boodaliboo**
### Intent of your Pull Request
adds a set of storm trooper armor to the game and allows you to buy this crate through cargo and open it if you have security access
(https://www.reddit.com/r/SS13/comments/65fjmm/star_wars_sprites_for_ss13/)
#### Changelog

:cl:  
rscadd: Added a new crate and stormertrooper armor
imageadd: added stormtrooper sprites
/:cl:

### CODE REVIEW

**Comment by MegaEmpirical** [comment]
Because pissing off Disney is a good idea.

**Comment by boodaliboo** [comment]
> Because pissing of Disney is a good idea.

we already are pissing of fox which is owned by disney and we have so many blatant references already light energy swords

**Comment by ynot01** [comment]
hello mister disney did you know a game that came out 16 years ago and looks the part with an active player count of <1000 is ripping off YOUR corporate ideas?? ill call the lawyers right now so we can sue the CEO of ss13

**Comment by boodaliboo** [comment]
fuck it failed

**Comment by boodaliboo** [comment]
what?

**Comment by boodaliboo** [comment]
it wont let me resolve conflicts so uh

**Comment by boodaliboo** [comment]
> it wont let me resolve conflicts so uh

if i can find a way to fix it i will

File: code/modules/clothing/suits/armor.dm [hunk 1]
```diff
@@ -269,3 +269,15 @@
...
+	
+/obj/item/clothing/suit/armor/stormtrooper
+	name = "Storm Trooper Armor"
+	desc = "Battle Armor from a long lost empire"
+	icon = icons/obj/clothing/starwars.dmi
+	icon_state = "startrooper"
+	body_parts_covered = CHEST|GROIN|LEGS|FEET|ARMS|HANDS
+	cold_protection = CHEST|GROIN|LEGS|FEET|ARMS|HANDS
+	heat_protection = CHEST|GROIN|LEGS|FEET|ARMS|HANDS
+	armor = list("melee" = 20 "bullet" = 20 "laser" = 60 "energy" = 20 "bomb" = 30 "bio" = 20 "rad" = 10 "fire" = 80, "acid" = 80)
```
Comment (missatessatessy) [inline code/modules/clothing/suits/armor.dm]: Commas between each thing eg "melee" = 20, "bullet"

**Comment by boodaliboo** [comment]
remaking this with correct icons

**Pull request closed without merging**